modules/markdown/parsers/createParser.ts

```diff
@@ -5,7 +5,7 @@ export const createParser = (
   rules: Record<string, MarkdownRule>,
   transform?: (ast: SingleASTNode[]) => SingleASTNode[],
 ) => {
-  const parse = parserFor(rules, { inline: true })
+  const parse = parserFor(rules)
```

Comment:
Dropping inline here messes up a lot of assumptions that are present in how markdown rendering works and breaks a ton of other formatting in the website and this isn't the right way to go about it.